Importer/EventListener/ErrorSubscriber.php

```diff
@@ -0,0 +1,186 @@
+<?php
+namespace Netdudes\ImporterBundle\Importer\EventListener;
+
+use Doctrine\DBAL\DBALException;
+use Netdudes\ImporterBundle\Importer\Event\Error\ApplicationErrorEvent;
+use Netdudes\ImporterBundle\Importer\Event\Error\CsvHeaderErrorEvent;
+use Netdudes\ImporterBundle\Importer\Event\Error\DataErrorEvent;
+use Netdudes\ImporterBundle\Importer\Event\Error\FileNotFountErrorEvent;
+use Netdudes\ImporterBundle\Importer\Event\Error\SaveErrorEvent;
+use Netdudes\ImporterBundle\Importer\Event\ImportEvents;
+use Netdudes\ImporterBundle\Importer\Interpreter\Exception\DateTimeFormatException;
+use Netdudes\ImporterBundle\Importer\Interpreter\Exception\InvalidEntityException;
+use Netdudes\ImporterBundle\Importer\Interpreter\Exception\LookupFieldException;
+use Netdudes\ImporterBundle\Importer\Interpreter\Exception\RowSizeMismatchException;
+use Netdudes\ImporterBundle\Importer\Interpreter\Exception\UnknownColumnException;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\Validator\ConstraintViolation;
+
+class ErrorSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @return array
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            ImportEvents::APPLICATION_ERROR => ['handleApplicationError'],
```

Comment:
Can we reduce the number of event types? Perhaps more based around what the data / consequence is?